android/mendixnative-release/build.gradle

```diff
@@ -0,0 +1,2 @@
+configurations.maybeCreate("default")
+artifacts.add("default", file('mendixnative-release.aar'))
```

Comment:
There are single quotes again here. Let's be consistent.

Reply:
done!